packages/project-build/src/db/build.ts

```diff
@@ -130,100 +123,89 @@ const createNewPageInstances = (): Build["instances"] => {
  *   2. when we clone a project
  * We create "prod" build when we publish a dev build.
  */
-export async function createBuild(
-  props: {
-    projectId: Build["projectId"];
-    env: "prod";
-    sourceBuild: Build | undefined;
-  },
-  context: AppContext,
-  client: Prisma.TransactionClient
-): Promise<void>;
-export async function createBuild(
+export const createBuild = async (
   props: {
     projectId: Build["projectId"];
-    env: "dev";
-    sourceBuild: Build | undefined;
-  },
-  context: AppContext,
-  client: Prisma.TransactionClient
-): Promise<void>;
-// eslint-disable-next-line func-style
-export async function createBuild(
-  props: {
-    projectId: Build["projectId"];
-    env: "dev" | "prod";
-    sourceBuild: Build | undefined;
   },
   _context: AppContext,
   client: Prisma.TransactionClient
```

Comment:
Context unused. Client maybe not needed? In the next function too.

Reply:
Its because of lazyness and because our security checks are not optimized, pretty sure we need to check rights inside Build. So we will use it 100% some day